integration/setup/setup.go

```diff
@@ -302,3 +212,85 @@ func installCredential(h *framework.Host) error {
 
 	return nil
 }
+
+func installHostPeerVPC(config Config) (string, error) {
+	log.Printf("creating Host Peer VPC stack")
+
+	os.Setenv("AWS_ROUTE_TABLE_0", env.AWSRouteTable0())
+	os.Setenv("AWS_ROUTE_TABLE_1", env.AWSRouteTable1())
+	os.Setenv("CLUSTER_NAME", env.ClusterID())
+	stackName := "host-peer-" + env.ClusterID()
+	stackInput := &cloudformation.CreateStackInput{
+		StackName:        aws.String(stackName),
+		TemplateBody:     aws.String(os.ExpandEnv(e2etemplates.AWSHostVPCStack)),
+		TimeoutInMinutes: aws.Int64(2),
+	}
+	_, err := config.AWSClient.CloudFormation.CreateStack(stackInput)
+	if err != nil {
+		return "", microerror.Mask(err)
+	}
+	err = config.AWSClient.CloudFormation.WaitUntilStackCreateComplete(&cloudformation.DescribeStacksInput{
+		StackName: aws.String(stackName),
+	})
+	if err != nil {
+		return "", microerror.Mask(err)
+	}
+	describeOutput, err := config.AWSClient.CloudFormation.DescribeStacks(&cloudformation.DescribeStacksInput{
+		StackName: aws.String(stackName),
+	})
+	if err != nil {
+		return "", microerror.Mask(err)
+	}
+	var vpcPeerID string
+	for _, o := range describeOutput.Stacks[0].Outputs {
```

Comment:
That looks like it can panic and we should ensure there is only exactly the one item we actually expect. I know this is just moved code. But just saying.